Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+def repositoryName = 'metrics-apb'
+def projectName = 'ci-apb-test'
```

Comment:
Should the name be randomized to avoid conflicts?

Reply:
I agree, this job could be running twice at the same time if you for some reason are testing 2 apbs, no?

Reply:
Hey guys, this sounds like a good idea, but I can't find solution for generating random strings that would not be too much complicated for this case.

I've a slightly different suggestion: to use global variable `currentBuild` and its property `id` or `startTimeInMillis` as a suffix for the project name. Wdyt?
https://qa.nuxeo.org/jenkins/pipeline-syntax/globals#currentBuild

Reply:
As long as it's unique it's ok. currentBuild.number looks good

Reply:
If this gets triggered from pull requests, then maybe `currentBuild.number` or `currentBuild.id` (same but stringified?) aren't sufficient, since each pull request will get its own job, and start at 1. This will raise the probability of a collision in low build numbers (probably most pull requests will trigger fewer than 10 builds?).

`startTimeInMillis` would probably be sufficient, but the situation that you're trying to work around is one where more than one build is running at the same time. You could hit rare cases where more than one build is triggered at the same time in milliseconds!